### ISSUE TITLE
"Error: Expected a pseudo-class or pseudo-element."  if embedded expressions is written just before the selector

### ISSUE THREAD

**Issue by yuki383** [stylelint/postcss-css-in-js#314]
### What steps are needed to reproduce the bug?

Run `stylelint ./file.js`

.stylelinttc
```
{
  "extends": [
    "stylelint-config-standard"
  ],
  "overrides": [
    {
      "files": [
        "**/*.js"
      ],
      "customSyntax": "@stylelint/postcss-css-in-js"
    }
  ]
}
```

file.js
```js
import styled, { css } from 'styled-components';

const Anything = styled.div`
  ${css`
    margin: 0;
  `}
  div {
  }
`;
```

### What did you expect to happen?

Stylelint runs successfully.

### What actually happened?

```
Error: Expected a pseudo-class or pseudo-element.
    at /Users/yuki/stylelint-error/hoge.js:4:3
    at Root._error (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:174:16)
    at Root.error (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/selectors/root.js:43:19)
    at Parser.error (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:740:21)
    at Parser.expected (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:1133:19)
    at Parser.pseudo (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:895:19)
    at Parser.parse (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:1084:14)
    at Parser.loop (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:1043:12)
    at new Parser (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/parser.js:164:10)
    at Processor._root (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/processor.js:53:18)
    at Processor._runSync (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/processor.js:100:21)
    at Processor.processSync (/Users/yuki/stylelint-error/node_modules/postcss-selector-parser/dist/processor.js:197:23)
    at normalize (/Users/yuki/stylelint-error/node_modules/stylelint/lib/rules/no-duplicate-selectors/index.js:170:26)
    at Array.map (<anonymous>)
    at /Users/yuki/stylelint-error/node_modules/stylelint/lib/rules/no-duplicate-selectors/index.js:70:56
    at /Users/yuki/stylelint-error/node_modules/postcss/lib/container.js:96:18
    at /Users/yuki/stylelint-error/node_modules/postcss/lib/container.js:55:18
    at Root.each (/Users/yuki/stylelint-error/node_modules/postcss/lib/container.js:41:16)
    at Root.walk (/Users/yuki/stylelint-error/node_modules/postcss/lib/container.js:52:17)
    at Root.walkRules (/Users/yuki/stylelint-error/node_modules/postcss/lib/container.js:94:19)
    at /Users/yuki/stylelint-error/node_modules/stylelint/lib/rules/no-duplicate-selectors/index.js:55:8
    at /Users/yuki/stylelint-error/node_modules/stylelint/lib/lintPostcssResult.js:114:8
    at Array.map (<anonymous>)
    at lintPostcssResult (/Users/yuki/stylelint-error/node_modules/stylelint/lib/lintPostcssResult.js:105:18)
    at lintSource (/Users/yuki/stylelint-error/node_modules/stylelint/lib/lintSource.js:103:8)
    at async /Users/yuki/stylelint-error/node_modules/stylelint/lib/standalone.js:214:27
    at async Promise.all (index 0)
    at async standalone (/Users/yuki/stylelint-error/node_modules/stylelint/lib/standalone.js:257:22)
```
